requirements.txt

```diff
@@ -1,10 +1,12 @@
 agraph-python>=101.0.7,<102
+ase
```

Comment:
Always add the version dependency. The main reason for this is reproducibility.

Reply:
Done